dashboard/src/components/partial/EntitiesList/EntitiesList.js

```diff
@@ -0,0 +1,143 @@
+import React from "react";
+import PropTypes from "prop-types";
+import gql from "graphql-tag";
+
+import TableList, {
+  TableListBody,
+  TableListEmptyState,
+} from "/components/TableList";
+
+import EntitiesListHeader from "./EntitiesListHeader";
+import EntitiesListItem from "./EntitiesListItem";
+
+export const arrayRemove = (arr, val) => {
```

Comment:
Not sure this was meant to be exported, albeit I would be okay with this and `arrayAdd` found their way into their own package.

Reply:
I'm totally unsure about what to do about lodash style utils, we could standardize on ramda, lodash, lodash/fp, or start a file full of our own.

Reply:
These don't need to be exported tho, will change.

Reply:
> I'm totally unsure about what to do about lodash style utils, we could standardize on ramda, lodash, lodash/fp, or start a file full of our own.

I lean toward ramada because as far as I'm concerned lodash/fp is undocumented. However, the downsides that immediately come to mind are:

- Each function being curried might be confusing and a bit daunting to contributors.
- I dislike that the amount of needless polymorphism in ramda this leads to (among other things) the code being littered with many type assertions, which, in a future where we use flow or typescript is needless.

... However, I digress this is worth a separate discussion.